rotkehlchen/tests/unit/decoders/test_makerdao_sai.py

```diff
@@ -2299,31 +2300,23 @@ def test_makerdao_sai_proxy_interaction(ethereum_transaction_decoder):
 @pytest.mark.vcr(filter_query_parameters=['apikey'])
 @pytest.mark.parametrize('ethereum_accounts', [['0xca482bCd75A6E0697aD6A1732aa187310b8372Df']])
 def test_makerdao_sai_cdp_migration(ethereum_transaction_decoder, ethereum_accounts):
+    """Data taken from
+    https://etherscan.io/tx/0x03620c6bf5edb7a7935953337ffcfac70d631cf2012d6c80d36828d636063318
+    """
+
```

Comment:
useless comment. 
```suggestion
    """

```

Reply:
ok, that type of comment was already in other test cases in the same files, so I thought to add it also here for consistency. I can remove, no problem